yaml2sbml/yaml2sbml.py

```diff
@@ -306,14 +319,26 @@ def _create_assignment(model: sbml.Model, assignment_id: str, formula: str):
         formula: str: contains the equation for the assignment rule
     """
     assignment_parameter = model.createParameter()
-    assignment_parameter.setId(assignment_id)
+
+    if assignment_parameter.setId(assignment_id):
+        raise RuntimeError(f'Unable to generate assignment with id '
+                           f'{assignment_id}. Invalid SBML identifier.')
+
     assignment_parameter.setName(assignment_id)
     assignment_parameter.setConstant(False)
     assignment_parameter.setUnits('dimensionless')
 
     assignment_rule = model.createAssignmentRule()
     assignment_rule.setVariable(assignment_id)
-    assignment_rule.setMath(sbml.parseL3Formula(formula))
+
+    sbml_formula = sbml.parseL3Formula(formula)
+
+    if sbml_formula is not None:
+        assignment_rule.setMath(sbml.parseL3Formula(formula))
+    else:
+        raise RuntimeError(f'Unable to generate assignment for formula '
+                           f'{formula}, libsbml can not parse the given '
+                           f'expression.')
```

Comment:
```suggestion
        raise RuntimeError('Unable to generate assignment for formula '
                           f'{formula}, libsbml can not parse the given '
                           'expression.')
```

Reply:
would also work, matter of style I guess

Reply:
I think I once had a style checker complain about this :D So slightly in favor of leaving it consistent :)